docs/app/documentation/component-docs/menu/menu-docs.component.html

```diff
@@ -15,3 +15,18 @@ <h2>Menu with Group</h2>
   </p>
 </component-example>
 <code-example [code]="menuGroupHtml" [language]="'HTML'"></code-example>
+
+<separator></separator>
+
+<h2>Menu with keyboard support</h2>
+<description>
+    Menu keyboard support can be added on every component, which is using <code>MenuComponent</code>. It simulates
```

Comment:
Keyboard support can be added on every component which uses the `<code>MenuComponent</code>`.